uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/directives/leave-request-popup.html

```diff
@@ -279,10 +279,10 @@ <h4 class="modal-title">
                   </span>
                 </ul>
               </div>
+            <div class="chr_leave-request-files_upload" ng-disabled="$ctrl.isMode('view')">
               <div
                 nv-file-drop
                 uploader="$ctrl.request.fileUploader"
-                class="chr_leave-request-files_upload"
                 ng-disabled="$ctrl.isMode('view')"
```

Comment:
Is this `ng-disabled` required? It is already present in the parent
`<div class="chr_leave-request-files_upload" ng-disabled="$ctrl.isMode('view')">`